.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
Yeah, I was referring specifically to the `--require`. Color does seem more user-preference.